homeassistant/components/light/mqtt_json.py

```diff
@@ -108,6 +127,14 @@ def __init__(self, hass, name, topic, qos, retain,
 
         self._flash_times = flash_times
 
+        self._templates = {}
+        for key, tpl in list(templates.items()):
+            if tpl is None:
```

Comment:
Actually the default values are here to ensure backward compatibility so if I should do anything that would be remove this check.